### DISCUSSION TITLE
How to host a website

### DISCUSSION THREAD

**Discussion by cambo2015** [appwrite/appwrite#1231] [Q&A] [answered]
Hi guys! I am coming from a nodejs background. I'm use to using git,npm and pm2. How would I show a html page with a domain? It's probably easier than I think. Thanks!🥳

**Comment by doricci** [comment, marked as answer]
Hey @cambo2015
Appwrite is a back end and doesn't do that for you
But you can use github page, netlify or gitlab page to host html/css/js page for free.

**Reply by cambo2015** [reply]
Thanks @doricci . This makes sense.
